consensus/src/chained_bft/liveness/local_pacemaker.rs

```diff
@@ -167,22 +163,24 @@ impl LocalPacemakerInner {
         counters::TIMEOUT_ROUNDS_COUNT.get();
         counters::TIMEOUT_COUNT.get();
 
-        Self {
+        let mut pacemaker = Self {
             time_interval,
             highest_committed_round,
             highest_qc_round,
             current_round,
             current_round_deadline: Instant::now(),
             time_service,
             new_round_events_sender,
-            local_timeout_sender,
-            external_timeout_sender,
+            timeout_sender,
             pacemaker_timeout_manager: PacemakerTimeoutManager::new(
                 pacemaker_timeout_quorum_size,
                 highest_timeout_certificates,
                 persistent_liveness_storage,
             ),
-        }
+        };
+        // To jump start the execution return the new round event for the current round.
```

Comment:
One day we'll get rid of bounded channels in places like this one and it'll be beautiful.
Btw this comment is obsolete, we're not returning anything here.